tasks/github_tasks.py

```diff
@@ -421,3 +422,33 @@ def assign_codereview_label(_, pr_id=-1):
     gh = GithubAPI('DataDog/datadog-agent')
     complexity = gh.get_codereview_complexity(pr_id)
     gh.update_review_complexity_labels(pr_id, complexity)
+
+
+@task
+def agenttelemetry_list_change_ack_check(_, pr_id=-1):
+    """
+    Change to `comp/core/agenttelemetry/impl/config.go` file requires to acknowledge
+    potential changes to Agent Telemetry metrics. If Agent Telemetry metric list has been changed,
+    the PR should be labeled with `need-change/agenttelemetry-governance` and follow
+    `Agent Telemetry Governance` instructions to potentially perform additional changes. See
+    https://datadoghq.atlassian.net/wiki/spaces/ASUP/pages/4340679635/Agent+Telemetry+Governance
+    for details.
+    """
+    from tasks.libs.ciproviders.github_api import GithubAPI
+
+    gh = GithubAPI('DataDog/datadog-agent')
+
+    pr_id = int(os.environ.get("PR_ID"))
```

Comment:
If you use the `pr_id` as input argument you don't need this
```suggestion
```

Reply:
I think it was failing without it (I had it originally). I will do one more experiment to make sure. Thank you.